src/fstalign.cpp

```diff
@@ -596,6 +596,10 @@ void write_stitches_to_nlp(vector<Stitching>& stitches, ofstream &output_nlp_fil
       }
 
       ref_tk = original_nlp_token;
+    } else if (stitch.comment.find("ins") == 0) {
+      logger->debug("an insertion was found for {} {}", stitch.hyptk, stitch.comment);
+      ref_tk = "";
+      stitch.comment = "ins(" + stitch.hyptk + ")";
```

Comment:
There's no `add_inserts` check now?

Reply:
Because we already left the loop here - https://github.com/revdotcom/fstalign/pull/48/files#diff-3620f9114e25f9cecbc7103539a463340918dd599878df0f52990f1cc7438b5fL574